src/Npgsql.GeoJSON/CrsMapExtensions.cs

```diff
@@ -0,0 +1,52 @@
+using System.Threading.Tasks;
+using Npgsql.GeoJSON.Internal;
+
+namespace Npgsql.GeoJSON;
+
+/// <summary>
+/// Extensions for getting a CrsMap from a database.
+/// </summary>
+public static class CrsMapExtensions
+{
+    /// <summary>
+    /// Gets the full crs details from the database.
+    /// </summary>
+    /// <param name="dataSource"></param>
+    /// <returns></returns>
+    public static async Task<CrsMap> GetCrsMapAsync(this NpgsqlDataSource dataSource)
```

Comment:
Maybe put these as static Load() methods on CrsMap rather than as extensions of NpgsqlDataSource?

Reply:
Possibly better yeah, though they do need a `using` at least